benchmarks/scripts/analyze.py

```diff
@@ -215,6 +215,12 @@ def iterate_case_dfs(args, callable):
 
     pattern = re.compile(args.R)
 
+    exact_values = {}
+    if args.args:
```

Comment:
To be sure will that also catch `-a` or only `--args`

Reply:
That catches `-a`